app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -74,7 +74,8 @@ class GesturesActivity : AppCompatActivity() {
   private fun initializeMap() {
     gesturesPlugin = binding.mapView.gestures
     gesturesManager = gesturesPlugin.getGesturesManager()
-
+    // enable map pitch at higher touch point angle (default 20.0f).
+    gesturesManager.shoveGestureDetector.maxShoveAngle = 60.0f
```

Comment:
Taking into account it's example activity - let's make UI for it and put it under three dots settings (located in top-right screen angle).

Reply:
The default that iOS allows to use is 45 degrees, any reason for using 20?

Reply:
Yeah. default comes from upstream. Do we want to enable the default value to 45 degrees for parity? (then I will also update in `GesturePluginImpl` as well).

Reply:
technically it's a breaking change in terms of behaviour. 
however if I get logic correctly we may consider it a bug and change it.

Reply:
I think it's more like a bug, and I think we can change the default to 45 degrees in GesturesPluginImpl.
As this is already configurable through `gesturesManager.shoveGestureDetector.maxShoveAngle`, would be good to document it in Gestures Guide(gesture customisation section)

Reply:
Yeah make sense. If we are changing the default to 45 degrees, then I think we don't need to add UI in `GestureActivity`. we can just update the gesture guide stating that pitch behavior can be changed with `maxShoveAngle`.  wdyt ?